tensorflow_lite_support/python/task/processor/proto/segmentations_pb2.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Segmentations protobuf."""
+
+from tensorflow_lite_support.cc.task.vision.proto import segmentations_pb2
```

Comment:
@kinaryml please move the python file to python/task/vision/proto/segmentations_pb2.py to correctly mirror the C++ proto path. I'm working on an internal change to update the image classifier and object detector protos so you don't need to worry about them.

Reply:
@khanhlvg Fixed